### DISCUSSION TITLE
movebarter

### DISCUSSION THREAD

**Discussion by baidang201** [NonceGeek/Move-Camp#251] [Amazing Things!]
一句话 IDEA：
基于move的nft交换平台

场景1:
在nft中，有些nft物品的价格太高，或者无法估价，只能互换各自需要的东西。

场景2:
在灾难响应的极端情况下，市场经济会失灵，它可以是蔬菜食品的可乐。

我的角色： move 合约开发，整体项目部署

人数需求（不包含发起者）：共需2人，1人 web3前端经验 1人Sui Move开发者

我的联系方式：tg smithwow

个人简介：5年区块链开发，有rust/nodejs经验

现阶段3-4人的团队较为合适,
期待加入, 一起交流和学习Sui

**Comment by xiegudong45** [comment]
已加入
前端项目代码仓库：https://github.com/movebarter/movebarter-frontend
合约代码仓库：https://github.com/movebarter/movebarter/tree/sui

**Comment by xiegudong45** [comment]
demo页面：
* 首页
![image](https://user-images.githubusercontent.com/30063330/218300440-75ec9892-072f-435d-910e-29eab4aa2228.png)
* NFT页面（创建、展示当前账户下的NFT）
![image](https://user-images.githubusercontent.com/30063330/218300478-cf96de12-2a08-41c9-999c-0f78084038c4.png)
* Order页面（创建、删除、交易Order）
![image](https://user-images.githubusercontent.com/30063330/218300486-cba72b58-e297-4c15-ae0d-3a55cbe6d192.png)
